WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailsActivity.kt

```diff
@@ -0,0 +1,49 @@
+package org.wordpress.android.ui.history
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import org.wordpress.android.R
+import org.wordpress.android.analytics.AnalyticsTracker
+import org.wordpress.android.analytics.AnalyticsTracker.Stat
+import org.wordpress.android.ui.history.HistoryListItem.Revision
+
+class HistoryDetailsActivity : AppCompatActivity() {
+    companion object {
+        const val KEY_HISTORY_DETAILS_FRAGMENT = "account-settings-fragment"
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.history_details_activity)
```

Comment:
```suggestion
        setContentView(R.layout.history_detail_activity)
```

Can we rename this layout `history_detail_activity` for consistency?